frontend/locale/de.json

```diff
@@ -17,65 +17,65 @@
   "pagination.older": "Ältere",
   "searchbar.label": "Suche nach Rezepten",
   "searchbar.mobile.label": "Suche",
-  "searchbar.placeholder": "Geben Sie einen Titel, Begriff oder eine Zutat ein.",
-  "file_widget.help_block": "Bitte laden Sie ein Bild des fertigen Gerichts hoch!",
+  "searchbar.placeholder": "Geben Sie einen Titel, ein Schlagwort oder eine Zutat ein.",
+  "file_widget.help_block": "Bitte laden Sie ein Foto des fertigen Gerichts hoch!",
   "select_widget.header": "Bitte wählen Sie ein {label} aus.",
   "widget_error.title": "Formularfehler!",
   "widget_error.message": "Bitte füllen Sie alle benötigten Felder aus.",
   "nav.create_recipe": "Rezept erstellen",
-  "nav.import": "Import Recipe",
-  "nav.grocery_list": "Rezept importieren",
+  "nav.import": "Rezept importieren",
+  "nav.grocery_list": "Einkaufsliste",
   "nav.login.title": "Login",
-  "nav.accountmenu.title": "Mein Account",
+  "nav.accountmenu.title": "Mein Benutzerkonto",
   "nav.accountmenu.logout": "Ausloggen",
-  "nav.accountmenu.admin": "Mein Account",
+  "nav.accountmenu.admin": "Mein Benutzerkonto",
   "nav.brand": "Open Eats",
-  "nav.news": "News",
-  "nav.recipes": "Rezepte",
+  "nav.news": "Neues",
```

Comment:
well i'm not sure if this is the correct translation...
- now it is like "is there something new"
- or should it be like "news in the newspaper"?

Reply:
It's ambiguous. It depends on how the user whats to use it. I personally would take it to me as   "news in the newspaper". But each person might have a different interpretation.

Reply:
i changed it back to "News", as it is also used as anglicism... and so it still has both meanings.